<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrawlMain</class>
 <widget class="QMainWindow" name="CrawlMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Crawl</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../crawl.qrc">
    <normaloff>:/crawl.png</normaloff>:/crawl.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWebEngineView" name="strollView">
       <property name="url">
        <url>
         <string>about:blank</string>
        </url>
       </property>
      </widget>
      <widget class="QWebEngineView" name="resultView">
       <property name="url">
        <url>
         <string>about:blank</string>
        </url>
       </property>
      </widget>
      <widget class="QWebEngineView" name="seedView">
       <property name="url">
        <url>
         <string>about:blank</string>
        </url>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="currentFetch"/>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="workLabel">
        <property name="text">
         <string>working</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="sendProgress">
        <property name="value">
         <number>0</number>
        </property>
        <property name="format">
         <string>%v of %m</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="addUrlLabel">
        <property name="text">
         <string>Seed Url</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="seedUrlEdit"/>
      </item>
     </layout>
    </item>
    <item row="7" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="sitemapCheck">
        <property name="text">
         <string>Site Map</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Add Seed</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="openmielke::CrawlButton" name="crawlButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>
Crawl</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cycleButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Cycle</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>488</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAgenda">
    <property name="title">
     <string>Crawl</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionSaveXML"/>
    <addaction name="actionSaveArado"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help...</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionLicense"/>
   </widget>
   <addaction name="menuAgenda"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionNewItem">
   <property name="text">
    <string>New Item</string>
   </property>
  </action>
  <action name="actionClearOld">
   <property name="text">
    <string>Clear Old</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="text">
    <string>Restart</string>
   </property>
  </action>
  <action name="actionLicense">
   <property name="text">
    <string>License</string>
   </property>
  </action>
  <action name="actionRevive">
   <property name="text">
    <string>Revive Expired</string>
   </property>
  </action>
  <action name="actionSaveXML">
   <property name="text">
    <string>Save as XML...</string>
   </property>
  </action>
  <action name="actionSaveArado">
   <property name="text">
    <string>Save as Arado...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebEngineView</header>
  </customwidget>
  <customwidget>
   <class>openmielke::CrawlButton</class>
   <extends>QPushButton</extends>
   <header location="global">src/crawl-button.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../crawl.qrc"/>
 </resources>
 <connections/>
</ui>
